plugins/scaffolder/src/components/TemplatePage/TemplatePage.tsx

```diff
@@ -57,8 +63,16 @@ export const createValidator = (
   rootSchema: JsonObject,
   validators: Record<
     string,
-    undefined | ((value: JsonValue, validation: FieldValidation) => void)
+    | undefined
+    | ((
+        value: JsonValue,
+        validation: FieldValidation,
+        context: { apiHolder: ApiHolder },
```

Comment:
Can make this `context?` otherwise it's a breaking change right?

Reply:
I was thinking that this is supplied by the environment, unconditionally, and you as a consumer won't want to have to check for the existence of this or not. So it makes sense to make a breaking change, I thought.

Interestingly, generating the API report, it does add a line `import { ApiHolder } from '@backstage/core-plugin-api';` at the top of it but NOT any usages. Hmmmmm

Reply:
Yeah I'm thinking that it's just a little ugly if you don't care about it, which most of them wont right?

And i'm also thinking that until we're deffo certain this is how we do it, maybe lets not break the API straight away to change it again later?